app/src/main/java/org/oppia/app/topic/TopicActivity.kt

```diff
@@ -39,7 +39,9 @@ class TopicActivity : InjectableAppCompatActivity(), RouteToQuestionPlayerListen
   }
 
   override fun routeToTopicPlayFragment() {
-    // TODO(#135): Change to play tab in this function.
+    (supportFragmentManager.findFragmentByTag(TOPIC_FRAGMENT_TAG) as TopicFragment).topicFragmentPresenter.setCurrentTab(
```

Comment:
This is a bit long and hence a bit hard to read. Suggest factoring the findFragmentByTag into a separate variable, e.g.:

```kotlin
val topicFragment = supportFragmentManager.findFragmentByTag(TOPIC_FRAGMENT_TAG) as TopicFragment
topicFragment.topicFragmentPresent.setCurrentTab(1)
```

Reply:
Resolved